src/main/java/seedu/address/model/person/SecLevel.java

```diff
@@ -0,0 +1,91 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Students's Secondary School Level in the address book.
+ */
+public class SecLevel {
```

Comment:
Just a question: did we specify that our target audience is tuition centres for secondary school students, or can our target audience cater to levels other than secondary school (eg. primary school or JC)? (A more general term for this would be Level).

Reply:
Good point. Let's bring this question up to our group discussion